### DISCUSSION TITLE
AppImage残留该怎么清理？

### DISCUSSION THREAD

**Discussion by earthmoon** [xiaoyifang/goldendict-ng#1033] [Q&A] [answered]
你好，

我使用了6.5.2 AppImage，系统为Debian12,添加了很多辞典。因为建立索引时，我重启了，现在打开goldendict就闪退。我该怎么清理索引，重新添加？

谢谢。

**Comment by xiaoyifang** [comment, marked as answer]
/home/[你的名称]/.cache/goldendict/index

**Reply by earthmoon** [reply]
好的，谢谢！
